orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -87,7 +87,7 @@ class DependentPipelineStarter implements ApplicationContextAware {
       strategy             : suppliedParameters.strategy == true
     ]
 
-    if (pipelineConfig.parameterConfig || !suppliedParameters.empty) {
+    if (pipelineConfig.parameterConfig || !suppliedParameters.isEmpty()) {
```

Comment:
This was just a bug, `suppliedParameters.empty` tried to look up the key `empty` in the map instead of calling the `isEmpty()` method.